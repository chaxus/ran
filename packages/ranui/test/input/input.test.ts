import Input from '@/components/input'

describe('input', () => {
  it('init input function', () => {
    const element = new Input()
    expect(element).toBeTruthy()
  })
  it('test input ui',()=>{
    new Input()
    const input = document.createElement('r-input')
    document.body.appendChild(input)
    expect(document.body).toMatchSnapshot();
  })
})
