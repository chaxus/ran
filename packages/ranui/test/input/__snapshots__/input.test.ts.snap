// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input test input ui 1`] = `
<body>
  <r-input />
</body>
`;
